Setup/autoupdate.sh

```diff
@@ -91,17 +91,18 @@ while true; do
             UPDATE=true
         fi
         if [ "$UPDATE" == true ]; then
-            curl -s -S -o ericomshield-setup.sh $ES_repo_setup
-            chmod +x ericomshield-setup.sh
             am_i_leader
             if [ "$AM_I_LEADER" == true ]; then
-                echo "Running Shield Setup (leader)"
-                echo "$(date): From autoupdate.sh Running Shield Setup (leader)" >>"$LOGFILE"
-                $ES_PATH/ericomshield-setup.sh
-            else
-                echo "Running Shield Setup no-deploy (I'm not the leader)"
-                echo "$(date): From autoupdate.sh Running Shield Setup no-deploy (I'm not the leader)" >>"$LOGFILE"
-                $ES_PATH/ericomshield-setup.sh -no-deploy
+                APPEND=""
+                if [ "$ES_DEV" == true ]; then
+                    APPEND="--dev"
+                fi
+
+                if [ "$ES_STAGING" == true ]; then
+                    APPEND="--staging"
+                fi
+
+                ./update.sh "$APPEND"
```

Comment:
As discussed, let's comment these lines and use the old method until update.sh is tested (basic)

Reply:
OK...
I run interesting tests with new Broker. Might be I will commit this changes....